frontend/src/components/pages/SearchReviews/ReviewThumbnail.tsx

```diff
@@ -34,6 +49,8 @@ const ReviewThumbnail = (props: ReviewThumbnailProps): React.ReactElement => {
         textStyle={["mobileBookTitle", "mobileBookTitle", "bookTitle"]}
         noOfLines={2}
         mt="4%"
+        onClick={() => history.push(`/reviews/${id}`)}
```

Comment:
Could you use the constant defined in the routes file here instead of hardcoding the path?

Reply:
For the constant, its defined as `export const DISPLAY_REVIEW_PAGE = "/reviews/:id";` For the id part of the constant, would I replace it with the actual id in a template literal?